public/index.html

```diff
@@ -139,7 +141,34 @@ <h2 class="sr-only">Apply for Health Care</h2>
     </div>
   </div>
 </div>
+<!-- HCA Application End -->
 
+<!-- Maintenance Page Start -->
+  <!--<div class="splash">
+    <div class="row">
+      <div class="small-12 columns">
+        <div class="pitch">
+          <h2 class="tagline"><span>Wednesday, June 29th, 2016 at 3:38 PM EST</span></h2>
+        </div>
+      </div>
+    </div>
+  </div>
+
+<div class="main home" role="main">
+
+  <div class="section main-menu">
+    <div class="row">
+      <div class="small-12 columns">
+        <div style="padding: 2em 0;">
+        <h3>We're sorry. The healthcare application is currently down while we fix a few things. We will be up and running as soon as we can.</h3>
```

Comment:
Maybe `We will be back up as soon as we can` instead of `up and running`? More clear?